libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -460,9 +460,10 @@ void AP_DDS_Client::on_topic(uxrSession* uxr_session, uxrObjectId object_id, uin
 
         subscribe_sample_count++;
         if (rx_dynamic_transforms_topic.transforms_size > 0) {
-            GCS_SEND_TEXT(MAV_SEVERITY_INFO,"Received tf2_msgs/TFMessage of length: %u",
-                          static_cast<unsigned>(rx_dynamic_transforms_topic.transforms_size));
-            // TODO implement external odometry to AP
+            #if AP_DDS_VISUALODOM_ENABLED
```

Comment:
should be left-aligned
```suggestion
#if AP_DDS_VISUALODOM_ENABLED
```

Reply:
Done 9273717a790723c241f7c2db3937b06977b0dcf7